docs/en/custom-font-setup.md

```diff
@@ -0,0 +1,80 @@
+# Custom Font Support for Label Maker
+
+Homebox label maker now supports external font files. **CJK (Chinese, Japanese, Korean) characters require a custom font** - the default bundled fonts only support Latin characters.
```

Comment:
Should probably just say "label maker supports" given that this will likely exist in the documentation long term, and it's very likely we would forget/never update this documentation unless there were a change later.